### PR TITLE
users: add Continental

### PR DESCRIPTION
**Pull request by waehlisch**
This adds the Continental use case based on the presentation at the RIOT Summit 2019.

### CODE REVIEW

File: _use_cases/continental.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+project: Keyless car sharing
+quote: RIOT is fun to hack. We value the modular design and the open-minded community.
```
Comment (emmanuelsearch) [inline _use_cases/continental.md:3]: Vincent told me he'd be fine with this quote:
"We use RIOT in our after-market product for connected car-sharing. RIOT's modular architecture helped us a lot to address the multiple challenges we faced, to design software and networking embedded on a low-power 32-bit microcontroller. Also: RIOT is fun to tinker with, and the RIOT community is very open minded!"

and this logo: https://www.carlogos.org/tire-logos/continental-logo-black-on-gold.png

Can you modify?

**Review by emmanuelsearch — CHANGES_REQUESTED** [review]

File: _use_cases/continental.md (hunk 1 above)
Comment (emmanuelsearch) [reply _use_cases/continental.md:3]: I see this has been merged faster than I could react... I'll PR changes.

Comment (waehlisch) [reply _use_cases/continental.md]: done.

**Review by emmanuelsearch — APPROVED** [review]
lgtm

**Pull request merged**